mlrun/api/db/sqldb/models/models_sqlite.py

```diff
@@ -87,7 +87,7 @@ class Tag(Base, mlrun.utils.db.BaseModel):
         obj_id = Column(Integer, ForeignKey(f"{table}.id"))
         obj_name = Column(
             String(255, collation=SQLCollationUtil.collation()),
-            ForeignKey(f"{table}.name"),
+            # ForeignKey(f"{table}.name"),
```

Comment:
this is problematic we want to keep the ForeignKey, you might want to pass `parent_key_name` to reference to as in `ArtifactV2` we have `key` and not `name`. or changing the `key` to `name` in the `ArtifactV2`